scripts/base/protocols/conn/main.zeek

```diff
@@ -312,6 +311,21 @@ event tunnel_changed(c: connection, e: EncapsulatingConnVector) &priority=5
 	c$tunnel = e;
 	}
 
+event new_connection(c: connection) &priority=100
+	{
+	set_conn(c, F);
+	}
+
+event connection_flipped(c: connection) &priority=5
+	{
+	# otherwise, set-conn has not been called yet. In that case we don't have to do anything
+	if ( c?$conn )
+		{
+		c$conn$local_orig=Site::is_local_addr(c$id$orig_h);
+		c$conn$local_resp=Site::is_local_addr(c$id$resp_h);
```

Comment:
```suggestion
		c$conn$local_resp = Site::is_local_addr(c$id$resp_h);
```

Eh, seems these are the only non-spaced assignments here.

Reply:
Thanks, I will fix this :)